packages/wrangler/e2e/dev.test.ts

```diff
@@ -1044,12 +1044,13 @@ describe("watch mode", () => {
 						`,
 					"public/index.html": dedent`
 								<h1>Hello Workers + Assets</h1>`,
+				});
+				await helper.seed({
 					"public2/index.html": dedent`
 								<h1>Hola Workers + Assets</h1>`,
 					"public2/about/index.html": dedent`
 								<h1>Read more about Workers + Assets</h1>`,
 				});
```

Comment:
I am curious why a separate seed helps?

Reply:
ah tbh its not necessary given we're using the absolute filepath now (it would have given it a different file creation/modified time and thus different hash)